README.rst

```diff
@@ -55,6 +55,10 @@ inputs:
 Changes
 =======
 
+Version 1.0.2 (2022-12-01)
+
+- Add `krenew` package to renew Kerberos tickets.
```

Comment:
It's not strictly speaking a package, because that one is called `kstart`. So perhaps say:

- Add `krenew` utility to allow renewing of Kerberos tickets for long-running jobs.